### PR TITLE
Modernize code sample

### PR DESCRIPTION
**Pull request by Youssef1313**
I don't see a good reason to have a class called `StudentClass` and a class called `Student` nested in it.
I've combined them in a `Student` class.

I've also renamed `ID` to `Id` to follow [Capitalization Conventions](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions#capitalizing-compound-words-and-common-terms).

### CODE REVIEW

File: snippets/csharp/concepts/linq/how-to-query-a-collection-of-objects_1.cs [hunk 1]
```diff
@@ -78,7 +74,7 @@ public class Program
     {
         public static void Main()
         {
-            StudentClass sc = new StudentClass();
+            var sc = new Student();
```
Comment (BillWagner) [inline snippets/csharp/concepts/linq/how-to-query-a-collection-of-objects_1.cs]: Consistent with the change I requested above, the `sc` object isn't needed, and the next line would be `Student.QueryHighScores(1,90);`

File: snippets/csharp/concepts/linq/how-to-query-a-collection-of-objects_1.cs [hunk 2]
```diff
@@ -1,66 +1,62 @@
...
+        // Helper method, used in GroupByRange.
         protected static int GetPercentile(Student s)
         {
             double avg = s.ExamScores.Average();
             return avg > 0 ? (int)avg / 10 : 0;
         }
 
-        
-
         public void QueryHighScores(int exam, int score)
```
Comment (BillWagner) [inline snippets/csharp/concepts/linq/how-to-query-a-collection-of-objects_1.cs]: After these changes, this class is more consistent if the `QueryHighScores` method is `static`.

Comment (Youssef1313) [reply snippets/csharp/concepts/linq/how-to-query-a-collection-of-objects_1.cs]:
```suggestion
        public static void QueryHighScores(int exam, int score)
```

File: snippets/csharp/concepts/linq/how-to-query-a-collection-of-objects_1.cs (hunk 1 above)
Comment (Youssef1313) [inline snippets/csharp/concepts/linq/how-to-query-a-collection-of-objects_1.cs]:
```suggestion
```

File: snippets/csharp/concepts/linq/how-to-query-a-collection-of-objects_1.cs [hunk 3]
```diff
@@ -78,7 +74,7 @@ public class Program
     {
         public static void Main()
         {
-            StudentClass sc = new StudentClass();
+            var sc = new Student();
             sc.QueryHighScores(1, 90);
```
Comment (Youssef1313) [inline snippets/csharp/concepts/linq/how-to-query-a-collection-of-objects_1.cs]:
```suggestion
            Student.QueryHighScores(1, 90);
```

**Comment by BillWagner** [comment]
Thanks for making the additional updates. I'll :shipit: now.

**Pull request merged**